tests/h/services/annotation_stats_test.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+import pytest
+
+from h.services.annotation_stats import AnnotationStatsService
+from h.services.annotation_stats import annotation_stats_factory
+
+
+class TestAnnotationStatsService(object):
+    @pytest.mark.usefixtures('user')
+    def test_user_annotation_counts_returns_count_of_annotations_for_user(self, svc, user, factories, db_session):
+        for i in range(3):
+            factories.Annotation(userid=user.userid, shared=True)
+        for i in range(2):
+            factories.Annotation(userid=user.userid, shared=False)
+        for i in range(4):
+            factories.Annotation(userid=user.userid, groupid='abc', shared=True)
+
+        results = svc.user_annotation_counts(user.userid)
+
+        assert results['public'] == 3
+        assert results['private'] == 2
+        assert results['group'] == 4
+
+    def test_annotation_count_returns_count_of_shared_annotations_for_group(self, svc, db_session, group, factories):
+        pubid = 'abc123'
+        for i in range(3):
+            factories.Annotation(groupid=pubid, shared=True)
+        for i in range(2):
+            factories.Annotation(groupid=pubid, shared=False)
+
+        assert svc.group_annotation_count(pubid) == 3
+
+    @pytest.fixture
```

Comment:
Ditto, this fixture isn't needed any more, because you're just using a static pubid of 'abc123'.